src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { NotificationMessage } from '../../types/NotificationMessage';
+import { Notification } from '../Notification/Notification';
+import { Loader } from '../Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loadingError, setLoadingError] = useState<boolean>(false);
+  const [noPeopleOnServer, setNoPeopleOnServer] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoadingError, setIsLoadingError] = useState<boolean>(false);
  const [IsNoPeopleOnServer, setIsNoPeopleOnServer] = useState<boolean>(false);
```

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Also pay attention for `isNoPeopleOnServer` always try have positive variable naming. For example `arePeoplePresent` || `arePeoplePresentOnServer`. Such naming more obvious